.azure-pipelines/template-skipvstest.yml

```diff
@@ -3,6 +3,13 @@ steps:
   - script: |
       set -ex
       tar_branch=origin/$(System.PullRequest.TargetBranch)
+      # Check if k8s master entrance script is changed
+      k8s_master_changed=$(git diff $tar_branch..HEAD --name-only | grep files/image_config/kubernetes/kubernetes_master_entrance.sh)
```

Comment:
>files/image_config/kubernetes/kubernetes_master_entrance.sh [](http://example.com/codeflow?start=73&length=59)

this is a regex, so `.` has special meaning.

Reply:
Use grep -F to fix